src/App.tsx

```diff
@@ -10,33 +12,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const handleTabSelected = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${tabs.find((tab) => tab.id === selectedTabId)?.title
```

Comment:
you can also write a separate func for this

Reply:
agree
Also, you can store tab in your state